src/main/java/com/capgemini/setrack/Model/Room.java

```diff
@@ -0,0 +1,67 @@
+package com.capgemini.setrack.Model;
+
+import com.capgemini.setrack.Model.Enums.RoomStatus;
+
+public class Room {
+
+    private RoomStatus roomStatus;
+    private String number;
```

Comment:
Waarom is de nummer een string? Als het een nummer is kan het int/long zijn, als het een naam is moet de variabele anders heten.